ui/components/app/snaps/snap-privacy-warning/index.scss

```diff
@@ -1,15 +1,19 @@
 .snap-privacy-warning {
   &__content {
-    max-height: 325px;
     overflow-y: auto;
 
+    @include screen-sm-max {
+      max-height: 312px; // fits for popover view
+    }
+
+
     &__scroll-button {
```

Comment:
Please keep the old logic for this button, as it is important that button is functional and visible on every possible screen/resolution within potential resolution scaling. It is important for this to be fully responsive because we cannot test this on every possible screen configuration and platform, etc.

If a user cannot click that button or scroll properly, they will not be able to continue (Accept) with snap installation and that problem is critical for us. So on very sensitive features like this, we pay more attention to that detail.

Reply:
I'm running build on top of this PR locally on MAC OS and when I resize a popup a little bit in chrome, I can see this:
![Screenshot 2023-08-24 at 20 20 31](https://github.com/MetaMask/metamask-extension/assets/13301024/424cfa23-070e-463a-9dd7-003fb268a1ac)

Previously, it was like this:
![Screenshot 2023-08-24 at 20 29 37](https://github.com/MetaMask/metamask-extension/assets/13301024/810989c2-dc71-4e32-8047-83c3788a07a0)
(Latest Flask build running on Firefox)

Reply:
Hey @david0xd, thanks for your thorough review! All great points. There are a couple of ways scrolling can be handled inside of a modal. Within the modal using a fixed height container or outside of the modal. The component library version was initially built with scrolling outside of the Modal for some of the reasons listed in this [UX Research post about multiple scrollbars and accessibility ](https://ux.stackexchange.com/questions/39298/research-on-scrolling-within-modal-web-pros-cons). We've gotten feedback that the expectation of this component is to have internal scrolling so I've made updates to retain both the styling, scroll button and internal scrolling. I've also ensured the scroll button and content can be seen at all viewports. This comes at the cost of multiple scroll bars at some awkward screen sizes but I think it will be very unlikely users are viewing MetaMask at very wide but very short screen sizes (< 600px height). I again want to thank you for the time put into reviewing and helping to improve the component library components through iterative testing and use cases.


https://github.com/MetaMask/metamask-extension/assets/8112138/699feb5b-03e4-4f99-b67c-44f9c512986b